app/src/main/java/nl/tudelft/pixelperfect/client/EventsMessage.java

```diff
@@ -0,0 +1,38 @@
+package nl.tudelft.pixelperfect.client;
+
+import com.jme3.network.AbstractMessage;
+import com.jme3.network.serializing.Serializable;
+
+/**
+ * The class that is sent by the server when it shares an event.
+ *
+ * @author Dmitry Malarev.
+ */
+@Serializable
+public class EventsMessage extends AbstractMessage{
+    private String event;
```

Comment:
Why doesn't the message have parameters of an event (e.g. type, summary, description et cetera)? By describing an Event with just a String you are making things unnecessarily complex.